src/main/java/com/cs544/project/domain/Student.java

```diff
@@ -25,7 +23,10 @@ public class Student extends Person {
     @Column(name = "ApplicantID")
     private String applicantID;
 
-    @ManyToOne
+    @ManyToOne(fetch = FetchType.LAZY)
```

Comment:
why lazy fetch?

Reply:
@bibangamba I am getting an error on the API due to not loading faculty on the student entity I was trying different techniques to fix it and it's one of them though not fixed yet. Could you help me to fix the issue?